src/test/suite/fixtures/recorded/languages/css/changeSelector.yml

```diff
@@ -0,0 +1,29 @@
+languageId: css
```

Comment:
Cool.  Worth also having a test where the cursor is inside the body of the statement rather than inside the selector itself